tests/Unit/Framework/ActionTesting.hpp

```diff
@@ -391,9 +390,9 @@ class MockArrayElementProxy {
  public:
   using Inbox = tuples::tagged_tuple_from_typelist<InboxTagList>;
 
-  MockArrayElementProxy(MockDistributedObject<Component>& local_algorithm,
+  MockArrayElementProxy(MockChareElement<Component>& mock_chare_element,
                         Inbox& inbox)
-      : local_algorithm_(local_algorithm), inbox_(inbox) {}
+      : mock_chare_element_(mock_chare_element), inbox_(inbox) {}
```

Comment:
Would `mock_distributed_objects` be clearer? I personally don't really know what `mock_chare_element` is and I'm worried that `element` will be confused as a DG element. Or since this is all the distributed objects, `all_mock_distributed_objects`?